From 4283f675f21d3923e200092cec6418014a03ad52 Mon Sep 17 00:00:00 2001
From: Johan Oudinet <johan.oudinet@gmail.com>
Date: Thu, 26 Dec 2019 17:06:34 +0100
Subject: [PATCH] fix ejabberdctl

Change default values so ejabberdctl run commands as ejabberd user.

Signed-off-by: Johan Oudinet <johan.oudinet@gmail.com>
---
 ejabberdctl.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ejabberdctl.template b/ejabberdctl.template
index a940c3df7..5166e5a27 100755
--- a/ejabberdctl.template
+++ b/ejabberdctl.template
@@ -15,7 +15,7 @@ SCRIPT_DIR=$(cd "${0%/*}" && pwd)
 ERL="{{erl}}"
 IEX="{{bindir}}/iex"
 EPMD="{{epmd}}"
-INSTALLUSER="{{installuser}}"
+INSTALLUSER=ejabberd
 
 # check the proper system user is used
 case $(id -un) in
-- 
2.20.1

